src/compiler/output-targets/dist-custom-elements/index.ts

```diff
@@ -34,15 +34,15 @@ import ts from 'typescript';
  * @returns an empty Promise which won't resolve until the work is done!
  */
 export const outputCustomElements = async (
-  config: d.Config,
+  config: d.ValidatedConfig,
```

Comment:
Minor, can we update the JSDoc description for this variable to better describe this is now a validated config? Something like:
```diff
- * @param config the user-supplied compiler configuration we're using
+ * @param config the validated compiler configuration we're using
```